yogstation/code/modules/surgery/shadowling/sling_organs.dm

```diff
@@ -0,0 +1,72 @@
+//Snowflake spot for putting sling organ related stuff
+/obj/item/organ/eyes/night_vision/alien/sling
+	name = "shadowling eyes"
+	desc = "The eyes of a spooky shadowling!"
+
+/obj/item/organ/internal/shadowtumor
+	name = "black tumor"
+	desc = "A tiny black mass with red tendrils trailing from it. It seems to shrivel in the light."
+	icon_state = "blacktumor"
+	w_class = 1
+	zone = "head"
+	slot = "brain_tumor"
+	var/health = 3
```

Comment:
What is this used for? I confused this for mob health, use a better name like ``organHealth`` if this relates to the organ. This isn't checked anywhere, the value just gets changed?

Reply:
![image](https://user-images.githubusercontent.com/25938722/42197024-de0c3f4c-7e4e-11e8-9918-3edfc5df1efc.png)
are you ok